common_logging/src/main/java/uk/gov/dhsc/htbhf/logging/event/CommonEventType.java

```diff
@@ -3,5 +3,6 @@
 import uk.gov.dhsc.htbhf.logging.EventType;
 
 public enum CommonEventType implements EventType {
-    APPLICATION_STARTED
+    APPLICATION_STARTED,
+    FAILED_EVENT
```

Comment:
As this is an EventType, should this just be `FAILED` rather than `FAILED_EVENT`?

Reply:
now FAILURE